server/plugin/api.go

```diff
@@ -178,6 +184,53 @@ func (p *Plugin) handleError(w http.ResponseWriter, r *http.Request, error *seri
 	}
 }
 
+// handleUnlinkProject unlinks a project
+func (p *Plugin) handleUnlinkProject(w http.ResponseWriter, r *http.Request) {
+	mattermostUserID := r.Header.Get(constants.HeaderMattermostUserIDAPI)
+
+	var project *serializers.ProjectDetails
+	decoder := json.NewDecoder(r.Body)
```

Comment:
It will be better to move this to a serializer if it is present and do we need to include validations here for the body of the api

Reply:
we don't have any serializer for it, but it's a good practice to keep the flow consistent for all APIs and also check for at least empty validation so created https://brightscout.atlassian.net/browse/MI-2052